lib/activegraphql/query.rb

```diff
@@ -0,0 +1,80 @@
+require 'httparty'
+require 'active_support/inflector'
+require 'activegraphql/support/fancy'
+
+module ActiveGraphQL
+  class Query < Support::Fancy
```

Comment:
A bit late to make a note about it ;) we talked about it a few days ago. Yeah, let's go with what we have for this first release